exporter/zipkinexporter/README.md

```diff
@@ -19,45 +19,38 @@ By default, this exporter requires TLS and offers queued retry capabilities.
 
 The following settings are required:
 
-- `endpoint` (no default): URL to which the exporter is going to send Zipkin trace data.
-- `format` (default = `JSON`): The format to sent events in. Can be set to `JSON` or `proto`.
-
-By default, TLS is enabled and must be configured under `tls:`:
-
-- `insecure` (default = `false`): whether to enable client transport security for
-  the exporter's connection.
-
-As a result, the following parameters are also required under `tls:`:
-
-- `cert_file` (no default): path to the TLS cert to use for TLS required connections. Should
-  only be used if `insecure` is set to false.
-- `key_file` (no default): path to the TLS key to use for TLS required connections. Should
-  only be used if `insecure` is set to false.
+- `endpoint` (no default): URL to which the exporter is going to send Zipkin trace data. For example: `http://localhost:9411/api/v2/spans`.
 
 The following settings are optional:
 
-- `defaultservicename` (default = `<missing service name>`): What to name
+- `format` (default = `json`): The format to sent events in. Can be set to `json` or `proto`.
+- `default_service_name` (default = `<missing service name>`): What to name
   services missing this information.
 
+To use TLS, specify `https://` as the protocol scheme in the URL passed to the `endpoint` property.
```

Comment:
Nit: It might be helpful to reference the `Advanced Configuration` section with this comment, so the reader knows there are more TLS settings available to configure.

Reply:
👍🏻 see the jaeger exporter's advance section https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/jaegerexporter#advanced-configuration

Reply:
:+1: Added link